src/tests/ftest/util/command_utils.py

```diff
@@ -430,6 +430,80 @@ def set_environment(self, env):
         self.env = env.copy()
 
 
+class TargetedExecutableCommand(CommandWithParameters):
+    """Executable command that uses run_pcmd() to run given command on a targeted host.
+    """
+
+    def __init__(self, namespace, command, host, path=""):
+        """Create a TargetedExecutableCommand object.
+
+        Args:
+            namespace (str): yaml namespace (path to parameters)
+            command (str): string of the command to be executed.
+            host (NodeSet): Single host to run the command on (targeted host).
```

Comment:
Couldn't this also be multiple hosts?

Reply:
Yes, although running ddb on multiple hosts would be a quite advanced test.